### PR TITLE
update angular to 13

### PR DESCRIPTION
**Pull request by behroozbc**
#847

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -74,7 +62,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```
Comment (rwwagner90) [inline angular.json:77]: Why was this removed?

File: angular.json [hunk 2]
```diff
@@ -123,18 +110,6 @@
               "projects/shepherd-tester/src/assets"
             ]
           }
-        },
-        "lint": {
```
Comment (rwwagner90) [inline angular.json:127]: Why are we removing all the linting?

File: projects/shepherd/tsconfig.lib.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
   },
   "angularCompilerOptions": {
     "annotateForClosureCompiler": true,
-    "enableIvy": false,
+    "compilationMode": "partial",
```
Comment (rwwagner90) [inline projects/shepherd/tsconfig.lib.json:22]: Can you please explain this change? Sorry I do not know a lot about angular.

**Comment by behroozbc** [comment]
I go way angular say in this guide https://update.angular.io/?v=12.0-13.0 
when run this command ```npx @angular/cli@13 update @angular/core@13 @angular/cli@13``` make this changes

File: projects/shepherd/tsconfig.lib.json (hunk 3 above)
Comment (behroozbc) [reply projects/shepherd/tsconfig.lib.json:22]: https://blog.angular.io/upcoming-improvements-to-angular-library-distribution-76c02f782aa4

File: angular.json (hunk 1 above)
Comment (behroozbc) [reply angular.json:77]: I this [version ](https://v11.angular.io/cli/build#:~:text=Deprecated%3A%20Deprecated%20since,instead%20of%20%27.js%27.)be come deprecated and in version 13 removed

File: angular.json (hunk 2 above)
Comment (behroozbc) [reply angular.json:127]: this related to this issue https://github.com/angular-eslint/angular-eslint/issues/715

**Comment by behroozbc** [comment]
@rwwagner90 hi
Can I help something to merge PR?
If I must change something please tell me

**Review by rwwagner90 — APPROVED** [review]

**Comment by rwwagner90** [comment]
Thanks for the explanation @behroozbc! Could you please rebase and fix the conflicts?

**Comment by behroozbc** [comment]
hi @rwwagner90  
conflicts is fixed

**Comment by rwwagner90** [comment]
Thanks @behroozbc!
